Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -93,10 +93,10 @@ reclaimPolicy: Delete
 allowVolumeExpansion: true
 ```
 
-If you've deployed the Rook operator in a namespace other than "rook-ceph",
-change the prefix in the provisioner to match the namespace
-you used. For example, if the Rook operator is running in the namespace "my-namespace" the
-provisioner value should be "my-namespace.rbd.csi.ceph.com".
+If you've deployed the Rook operator in a namespace other than `rook-ceph`,
```

Comment:
please update other instance of "

like: "Example Output: kubectl get pvc"

Reply:
This looks resolved now, yes? I think we can merge this